OBSDOCS-2307: Port the Log collection and forwarding chapter to 6.x (#94412)
openshift/openshift-docs: modules/forwarding-logs-to-amazon-s3-endpoint.adoc

metadata:
  name: <log_forwarder_name>
spec:
  serviceAccount:
    name: <service_account_name> # <1>
  outputs:
  - name: s3bucket # <2>
    type: s3 # <3>
    s3:
      authentication:
        type: awsAccessKey
        awsAccessKey:
          keyId:
            key: aws_access_key_id 
            secretName: aws-secret # <4>
          keySecret:
            key: aws_secret_access_key
            secretName: aws-secret
      bucket: <bucket_name> # <5>
      keyPrefix: 's3{.log_type||"missing"}' # <6>
      region: us-east-1     
      url: https://s3.us-east-1.amazonaws.com # <7>
  pipelines:
  - name: pipe1
    inputRefs:
    - application
    outputRefs:
    - s3bucket
----
<1> The name of your service account.
<2> Specify a name for the output.
<3> Specify the `s3` type.
<4> Specify the name of the secret that contains your AWS credentials.
<5> Bucket specifies the S3 bucket name where logs will be stored.
<6> Specify the  S3 key prefix for log objects. The value can be a combination of static and dynamic values consisting of field paths separated by "||". Encase dynamic values in single curly brackets "{}". Dynamic values must be followed by a static fallback value.

> <7> Specify the custom S3-compatible endpoint URL. If not specified, the default AWS S3 endpoint will be used.

Reviewer: ```suggestion
<7> Specify the custom S3-compatible endpoint URL. If you do not specify this, the system uses the default AWS S3 endpoint.
```
Passive voice. Also, shouldn't use the future tense.

Author: done

  namespace: openshift-logging
spec:
  serviceAccount:
    name: <service_account_name> #<1>
  outputs:
  - name: s3bucket #<2>
    type: s3 #<3>
    s3:
      authentication:
        type: awsAccessKey
        awsAccessKey:
          keyId:
            key: aws_access_key_id 
            secretName: cw-secret #<4>
          keySecret:
            key: aws_secret_access_key

>             secretName: cw-secret

Author: done
